examples/GroveButton/GroveButton.ino

```diff
@@ -19,21 +16,36 @@ Licensed under the MIT license. See LICENSE file in the project root for full li
 
 //A 512 offset is required for sub-platforms.  516 corresponds to digital pin 4, or D4.
 #define BUTTON_PIN 516
+void setup();
```

Comment:
These should not exist here.

Reply:
I was trying to maintain a consistency across all files. These declaration was already there in all files, so I added. Let me know, what you prefer. Should I remove in all files?

Reply:
For example in RotaryAngle, LCD examples.